a.txt

```diff
@@ -1,2 +1,3 @@
 line 1 in a.txt
 some another line that is added in github for a new commit
+new line in file a.txt from the new user account
```

Comment:
The first letter of the sentence should be upper case. Please correct it and make another request.